test/apps/test_fuzzy_match.py

```diff
@@ -86,12 +97,12 @@ def get_fuzzy_matches(self):
 
     try:    
         runner = TestRunner(args.ucx_info, args.verbose)
-        expected_list = [{'UCX_LOF_LEVEL' : ['UCX_LOG_LEVEL']}, 
-                         {'UCX_LOF_LEVEL' : ['UCX_LOG_LEVEL'], 'UCX_MOFULE_D' : ['UCX_MODULE_DIR', 'UCX_MODULES']},
-                         {'UCX_SOME_VAR' : [], 'UCX_SOME_VAR2' : [],  'UCX_SOME_VAR3' : [],  'UCX_SOME_VAR4' : []},
-                         {'UCX_SOME_VAR' : [], 'UCX_MOFULE_D' : ['UCX_MODULE_DIR', 'UCX_MODULES'], 'UCX_SOME_VAR2' : [], 'UCX_LOF_LEVEL' : ['UCX_LOG_LEVEL']},
-                         {'UCX_RC_VERBS_RX_MAX_BUF' : ['UCX_RC_VERBS_TX_MAX_BUFS', 'UCX_RC_VERBS_RX_MAX_BUFS', 'UCX_UD_VERBS_RX_MAX_BUFS']},
-                         {'UCX_RLS' : ['UCX_TLS']}]
+        expected_list =  [{'results' : {'UCX_LOF_LEVEL' : ['UCX_LOG_LEVEL']}, 'ib_required': False}, 
```

Comment:
WDYT of the following refactoring (simplify).
1. Rename `expected_list` to something like `test_cases` or `fuzzy_matches` (at least remove `_list`).
2. Revert to the "old" structure of the dicts in the list
3. Split to 2 lists, e.g `test_cases`, `ib_test_cases`.
4. Then can have:

```
def run(self, test_case, needs_ib=False):
    ...
```
```
for test_case in test_cases:
    runner.run(test_case)

for test_case in ib_test_cases:
    runner.run(test_case, needs_ib=True)
```